specs/unified_kernel_image.md

```diff
@@ -130,7 +133,7 @@ the measurements are interleaved: section name followed by section
 data, followed by the next section name and its section data, and so
 on.
 
-If multiple `.dtb` sections are present, they shall be measured in the
+If multiple `.dtbauto` sections are present, they shall be measured in the
 order they appear in the PE file.
```

Comment:
this is not what happens in sd-stub: we only measure the section we actually use. and i think that makes sense.

can you update the wording to say that only the `.dtb` or `.dtbauto` section we actually use is measured.

Reply:
This is not true? I just checked the systemd source code  and `measure_sections()` before any device tree selection (`install_embedded_devicetree`).  What am I missing here? 

I think how I am reading the code  the first `.dtbauto` section found is what is always measured.  Not the `.dtbauto` section that is used. Which sounds completely wrong?